Documentation/general/tool-nuget-package-format.md

```diff
@@ -0,0 +1,26 @@
+Tool NuGet package format
+-------------------------------------------
+
+Use `dotnet pack` with `PackAsTool` property is the most straight forward way. However, if your build process is highly customized and `dotnet publish` cannot create a proper distribution for you. You can create tool NuGet package using nuspec and explicitly place assets in NuGet package under the following rules.
+
+- Tool NuGet package does not have any other folder under root like _/lib_, _/content_, it only has _/tools_ folder.
+- Under _/tools_ folder, the subfolders are structured with pattern _target framework short version/RID_. For example, tool assets targeting .NET core framework V2.1 and portable should be under _tools/netcoreapp2.1/any_. Let's call assets under every _tools/target framework short version/RID_ combination per TFM-RID assets.
+    - There is a DotnetToolSettings.xml for every per TFM-RID assets.
+    - The package type is DotnetTool
+    - Per TFM-RID assets should have all the dependency the tool required to run. TFM-RID assets should be able to run using xcopy to another machine with correct runtime version and RID environment. For portable app, there is runtimeconfig.json for every per TFM-RID assets.

```

Comment:
- Each set of TFM-RID assets should have all the dependencies the tool requires to run. The TFM-RID assets should be work correctly after being copied via `xcopy` to another machine, assuming that machine has the correct runtime version and RID environment. 

I have a question on the last sentence. Should this be a separate bullet point? Should this "must be" instead of "there is"

For portable app, there is runtimeconfig.json for every per TFM-RID assets.

Reply:
Yes, "must be"